sdks/python/apache_beam/examples/snippets/util.py

```diff
@@ -0,0 +1,35 @@
+import argparse
```

Comment:
maybe add a util_test.py for the methods here?

Reply:
Tests added